data/Makefile.am

```diff
@@ -1,3 +1,7 @@
 iconoverridesdir = $(datadir)/eos-application-tools/icon-overrides
 
 dist_iconoverrides_DATA = eos-icon-overrides.ini
+
+flatpaksdir = $(datadir)/eos-application-tools/flatpaks
+
+dist_flatpaks_DATA = $(datadir)/eos-application-tools/autoinstall.conf
```

Comment:
Shouldn’t this path be `dist_flatpaks_DATA = autoinstall.conf`?

Reply:
Yup, brainfart moment. I've fixed up both comments now.